craft_providers/multipass/multipass.py

```diff
@@ -50,10 +50,10 @@ def _run(
         self,
         command: List[str],
         *,
-        check=True,
+        check: bool,
```

Comment:
I don't mind this change, but the PR should probably mention why

Reply:
Agreed, fixed.